app/Repositories/Metric/AbstractMetricRepository.php

```diff
@@ -58,6 +58,11 @@ protected function getTableName()
         $connection = $this->config->get('database.connections.'.$driver);
         $prefix = $connection['prefix'];
 
-        return $prefix.'metrics';
+        $return = (object) array(
+                        'metrics_base' => $prefix.'metrics',
```

Comment:
I'm not a fan of concatenation. Especially when we have possible PHP7 optimisations we can have here.